studies/models.py

```diff
@@ -512,6 +512,7 @@ def notify_administrators_of_submission(self, ev):
                     name=SiteAdminGroup.LOOKIT_ADMIN.name
                 ).user_set.values_list("username", flat=True)
             ),
+            reply_to=[self.request.user.username],
```

Comment:
self in this context is the study, so it doesn't have a request like a view would - it looks like we have the user making the request as `ev.kwargs.get("user")`

Reply:
Addressed in commit https://github.com/lookit/lookit-api/pull/671/commits/854fafc171a64f0fc335e93db276031431e6e49e.